test/Misc/serialized-diagnostics-batch-mode-nonspecific-error.swift

```diff
@@ -0,0 +1,26 @@
+// To avoid redundant diagnostics showing up in Xcode, batch-mode must suppress diagnostics in
+// non-primary files.
+// But if the only  errors are suppressed ones, a nonspecific error must be emitted
```

Comment:
As mentioned above, I think I was misleading about "suppression". This is something important even for errors that *do* get emitted, just for another file in the batch.

Reply:
Changing the verbiage.